src/resourcepacks/ResourcePackManager.php

```diff
@@ -58,6 +58,9 @@ class ResourcePackManager{
 	 */
 	private array $encryptionKeys = [];
 
+	/** @var string[] */
```

Comment:
```suggestion
	/** @var array<string, string> */
```

Reply:
This syntax should only be used in @phpstan-var.  Regular @var should be PSR-5 compliant.